src/server/components/dashboard/listsItems/controllers.ts

```diff
@@ -75,19 +75,25 @@ export async function listItemGetController(req: Request, res: ListItemRes): Pro
     requestedChanges = jsonData?.requestedChanges;
   }
 
-  const actionButtons: Record<Status, string[]> = {
-    NEW: ["publish", "request-changes", "unpublish"],
-    OUT_WITH_PROVIDER:
-      listItem.isPublished || listItem.isAnnualReview ? ["unpublish"] : ["publish", "request-changes", "unpublish"], // unpublish here should be replaced with archive
-    EDITED: [listItem.isPublished ? "update-live" : "update-new", "request-changes", "unpublish"], // unpublish here should be replaced with archive
-    PUBLISHED: ["request-changes", "unpublish"],
-    UNPUBLISHED: ["publish", "request-changes", "remove", "archive"],
-    CHECK_ANNUAL_REVIEW: ["request-changes", "unpublish", "publish"],
-    ANNUAL_REVIEW_OVERDUE: ["unpublish"],
+  const publishingStatus = getPublishingStatus(listItem);
+
+  const actions: Record<Action, boolean> = {
```

Comment:
Most of the status display very similar actions. `isPublished` is more likely to influence what is shown/hidden. I've flipped this object so it maps action to "what causes it to display".